cylc/flow/task_job_mgr.py

```diff
@@ -942,7 +942,7 @@ def _run_job_cmd(
                 cmd = construct_ssh_cmd(
                     cmd, platform, host
                 )
-            for itask in sorted(itasks, key=lambda itask: itask.identity):
+            for itask in sorted(itasks, key=lambda jtask: jtask.identity):
```

Comment:
Same as previous point: Although I think most of us learn that `itask` is our convention for `task_proxy` I think it would be more helpful to call it `task_proxy` than `jtask`. Unlike the previous version I think that it might require a newline, so I will be slightly less keen on the change.

Reply:
IMHO for a lambda function it would be ok to use a single letter variable, it should be relatively obvious what it's referring to in this case? (And I find `jtask` a bit weird)
```suggestion
            for itask in sorted(itasks, key=lambda i: i.identity):
```

Reply:
>  (And I find jtask a bit weird)

Who let the electrical engineers in?